rules/windows/privilege_escalation_printspooler_malicious_driver_file_changes.toml

```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2021/07/06"
+maturity = "production"
+updated_date = "2021/07/06"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects attempts to exploit privilege escalation vulnerabilities related to the Print Spooler service. For more
+information refer to the following CVE-2021-34527 and verify that the impacted system is investigated.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential PrintNightmare File Modification"
+references = [
+    "https://msrc.microsoft.com/update-guide/vulnerability/CVE-2021-34527",
+    "https://github.com/afwu/PrintNightmare"
+]
+risk_score = 73
+rule_id = "5e87f165-45c2-4b80-bfa5-52822552c997"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+file where process.name : "spoolsv.exe" and 
```

Comment:
Fyi for others,, this covers us for sysmon. The registry rule won't hit due to a schema issue

Reply:
added as a note comment in the query